client/state/comments/reducer.js

```diff
@@ -288,7 +290,19 @@ export const totalCommentsCount = createReducer(
 export const errors = createReducer(
 	{},
 	{
-		[ COMMENTS_ERROR ]: ( state, action ) => {
+		[ COMMENTS_RECEIVE_ERROR ]: ( state, action ) => {
```

Comment:
should this also remove the key from state on a successful write? or do you think it doesn't matter?

Reply:
Probably a good idea, but I've think improving receive errors is probably outside the scope of this PR. They could also benefit from some tidying in the future though.

Reply:
Oh wait - @samouri, do you mean removing write errors from `state.comments.errors`?

Reply:
yep